### PR TITLE
Add Support Hero /  Issue trackers, Documentation

### PR DESCRIPTION
**Pull request by gemini133**

### CODE REVIEW

**Comment by gemini133** [comment]
example site https://help.importify.com/

File: src/technologies/s.json [hunk 1]
```diff
@@ -4274,6 +4274,18 @@
     "icon": "Oracle.png",
     "website": "http://oracle.com/solaris"
   },
+  "Super Hero": {
```
Comment (nurbek91) [inline src/technologies/s.json:4277]: Change tech name to Support Hero.

File: src/technologies/s.json [hunk 2]
```diff
@@ -4274,6 +4274,18 @@
...
+    "cats": [
+      4,
+      13
+    ],
+    "description": "Super Hero is a knowledge base solution to reduce inbound support requests.",
```
Comment (nurbek91) [inline src/technologies/s.json:4282]: Support Hero is a knowledge base solution to reduce inbound support requests.

File: src/technologies/s.json [hunk 3]
```diff
@@ -4274,6 +4274,18 @@
...
+    "html": [
+      "<html[^>]+?data-ng-app=\"ngSupportHero\""
+    ],
+    "icon": "superhero.png",
```
Comment (nurbek91) [inline src/technologies/s.json:4286]: Support Hero.png

Comment (nurbek91) [reply src/technologies/s.json:4286]: Add pricing:
```
"saas": true,
    "pricing": [
      "mid",
      "recurring"
    ],
```

File: src/technologies/s.json [hunk 4]
```diff
@@ -4274,6 +4274,18 @@
     "icon": "Oracle.png",
     "website": "http://oracle.com/solaris"
   },
+  "Super Hero": {
+    "cats": [
+      4,
+      13
+    ],
+    "description": "Super Hero is a knowledge base solution to reduce inbound support requests.",
+    "html": [
```
Comment (nurbek91) [inline src/technologies/s.json:4283]: Lets remove` html` and use `js` and` dom`:
```
"js": {
      "supportHeroWidget": "",
      "supporthero": ""
    },
"dom": "a[href*='.supporthero.io/'][target='_blank']"
```

**Review by nurbek91 — CHANGES_REQUESTED** [review]
### Hey there thanks for PR. Some changes will be required
### examples:
**external link:**
https://home.by.me
https://www.freightos.com/
https://www.fluentify.com/
**Where does the link lead**:
https://fluentify.supporthero.io/login/signIn
**like widget:** 
https://www.toner.fr/
https://fluentify.supporthero.io/login/signIn
https://www.golfgenius.com/
https://cengage.com.au/

**Comment by gemini133** [comment]
thanks for your help. all resolved!

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**